### DISCUSSION TITLE
https://github.com/symfony/webpack-encore-pack not available anymore

### DISCUSSION THREAD

**Discussion by gewooneendeveloper** [symfony/symfony#42700] [Q&A] [answered]
Currently we are moving to SF 4.4, but just before we finish the current build is failing because the repo 
https://github.com/symfony/webpack-encore-pack is not available anymore.

Suggested fix, republish the https://github.com/symfony/webpack-encore-pack and source so SF 3 with webpack-encore-pack can be updated.

best regards,
Andreas

**Comment by Nek-** [comment]
Hello @andreaswarnaar , sorry to hear you experiment errors about the suppression of this package.

The way to fix your issue is not to republish the pack (which is deprecated for a long time), but you should install the bundle:

```
composer require symfony/webpack-encore-bundle
```

This bundle is (obviously) compatible with Symfony 4.4.

**Reply by 537mfb** [reply]
try editing composer.json and replace webpack-encore-pack with webpack-encore-bundle: `"symfony/webpack-encore-bundle": "^1.12",`

then run `composer update`

**Comment by localheinz** [comment, marked as answer]
@andreaswarnaar 

Here is what I did to remove the dependency on `symfony/webpack-encore-pack`:

### Require dependencies of `symfony/webpack-encore-pack`

Run

```
$ composer require symfony/asset
```

to require the only dependency of `symfony/webpack-encore-pack`.

### Remove `symfony/webpack-encore-pack`

Run

```
$ composer remove symfony/webpack-encore-pack
```

to remove the meta-package.

### Revert changes observed by unconfiguring `symfony/webpack-encore-pack`

When using `symfony/flex`, removal of `symfony/webpack-encore-pack` will also unconfigure `symfony/webpack-encore-pack`:

```shell
Symfony operations: 1 recipe (edbfcedc08e05d09c813edc1590848b4)
  - Unconfiguring symfony/webpack-encore-pack (>=1.0): From github.com/symfony/recipes:master
```

Observe the changes to `.gitignore`:

```diff
diff --git a/.gitignore b/.gitignore
index d5878a93f..33793322a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -13,13 +13,6 @@
 ###> symfony/phpunit-bridge ###
 .phpunit
 ###< symfony/phpunit-bridge ###
-###> symfony/webpack-encore-pack ###
-/node_modules/
-/public/build/
-npm-debug.log
-yarn-error.log
-###< symfony/webpack-encore-pack ###
-
 ###> liip/imagine-bundle ###
 /public/media/cache/
 ###< liip/imagine-bundle ###
```

Revert the changes applied to `.gitignore`, then you are good to go.

**Reply by hacfi** [reply]
Thanks a lot @localheinz - that was really useful! An old `symfony/flex`version caused some other errors additionally and I had to either run `rm -r vendor` or `composer require symfony/flex:"^1.5.0"` (it wasn't really clear what did the trick)
